__tests__/app.test.js

```diff
@@ -111,4 +111,38 @@ describe('/api/articles', () => {
             expect(response.body.articles).toBeSortedBy('created_at', {descending: true})
         })
     })
+})
+describe('api/articles/:article_id/comments', () => {
+    test('GET:200 sends all of the comments from an article to the client with all the desired properties sorted by recency', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.comments).toBeSortedBy('created_at', {descending: true})
+            response.body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number');
+                expect(typeof comment.created_at).toBe('string');
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
```

Comment:
Ideally we want to check the ID here is 3

Reply:
We also want a test here for 
Status 200, valid ID, but has no comments responds with an empty array of comments